src/clr-core/button/button.element.scss

```diff
@@ -1,35 +1,55 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+
 @import './../styles/mixins/utils';
+@import './../styles/mixins/mixins';
 
-// TODO: progress spinner CSS deprecated, will be replaced by cds-progress
+// TODO: progress spinner CSS deprecated, will be replaced by cds-circular-progress
 @import './../../clr-angular/progress/spinner/variables.spinner';
 @import './../../clr-angular/image/icons.clarity';
 @import './../../clr-angular/progress/spinner/spinner.clarity';
 
 :host {
-  --box-shadow-color: var(--clr-color-action-500, #{$clr-color-action-500});
-  --border-radius: #{$clr_baselineRem_3px};
-  --border-color: var(--clr-color-action-600, #{$clr-color-action-600});
-  --color: var(--clr-color-neutral-0, #{$clr-color-neutral-0});
-  --background: var(--clr-color-action-600, #{$clr-color-action-600});
+  --box-shadow-color: #{$cds-token-color-action-900};
+  --border-radius: #{$cds-token-global-border-radius};
+  --border-width: #{$cds-token-global-border-width};
+  --border-color: #{$cds-token-color-action-600};
+  --color: #{$cds-token-color-neutral-0};
+  --background: #{$cds-token-color-action-600};
+  --height: #{$clr_baselineRem_1_5};
```

Comment:
When i was using LHEs, it was shrinking the height of the button. That got left in there. It may no longer be needed but I'm also going to circle back to the LHEs to see if I can get them going.

Reply:
I've verified that `--height` is not needed.